acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -132,13 +132,17 @@ export default async function init(element) {
     sampleRate: 1,
     tags: 'Cat=DxDC_Frictionless,origin=milo',
   };
+  // LANA
+  window.dcwErrors = [];
   if (!DC_WIDGET_VERSION) {
     DC_WIDGET_VERSION = DC_WIDGET_VERSION_FALLBACK;
     window.lana?.log(`DC WIDGET VERSION IS NOT SET, USING FALLBACK VERSION: ${DC_WIDGET_VERSION_FALLBACK}`, lanaOptions);
+    dcwErrors.push(`DC WIDGET VERSION IS NOT SET, USING FALLBACK VERSION: ${DC_WIDGET_VERSION_FALLBACK}`);
```

Comment:
should these reference `window.dcwErrors` ?

Reply:
yes

Reply:
I meant to ask whether this line should be changed to include `window.`. Probably only a lint issue.